src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -130,6 +152,52 @@ struct RPPTensorFunctionMetaData
 };
 #endif // GPU_SUPPORT
 
+inline float rpp_host_math_inverse_sqrt_1(float x)
```

Comment:
please add a comment with the link to the the paper taking this algorithm

Reply:
Added citation - Lomont, C., 2003. FAST INVERSE SQUARE ROOT. [online] lomont.org. Available at: <http://www.lomont.org/papers/2003/InvSqrt.pdf>